monkey/infection_monkey/master/mock_master.py

```diff
@@ -93,32 +93,35 @@ def _fingerprint(self):
 
         self._puppet.fingerprint("HTTPFinger", machine_3)
         self._telemetry_messenger.send_telemetry(ScanTelem(machine_3))
-        logging.info("Finished running fingerprinters on potential victims")
+        logger.info("Finished running fingerprinters on potential victims")
 
     def _exploit(self):
-        logging.info("Exploiting victims")
+        logger.info("Exploiting victims")
         result, info, attempts = self._puppet.exploit_host(
             "PowerShellExploiter", "10.0.0.1", {}, None
         )
+        logger.info(f"Attempts for exploiting {attempts}")
         self._telemetry_messenger.send_telemetry(
             ExploitTelem("PowerShellExploiter", self._hosts["10.0.0.1"], result, info, attempts)
         )
 
         result, info, attempts = self._puppet.exploit_host("SSHExploiter", "10.0.0.3", {}, None)
+        logger.info(f"Attempts for exploiting {attempts}")
         self._telemetry_messenger.send_telemetry(
             ExploitTelem("SSHExploiter", self._hosts["10.0.0.3"], result, info, attempts)
         )
-        logging.info("Finished exploiting victims")
+        logger.info("Finished exploiting victims")
 
     def _run_payload(self):
-        logging.info("Running payloads")
+        logger.info("Running payloads")
         # TODO: modify what FileEncryptionTelem gets
         path, success, error = self._puppet.run_payload("RansomwarePayload", {}, None)
         self._telemetry_messenger.send_telemetry(FileEncryptionTelem(path, success, error))
-        logging.info("Finished running payloads")
+        logger.info("Finished running payloads")
 
     def terminate(self) -> None:
         logger.info("Terminating MockMaster")
+        exit(1)
```

Comment:
Terminating the master component returns control to the agent/main component. It does not cause the process to exit.

Reply:
This isn't needed, my bad. I got confused since both masters were running initially and nothing was stopping. Dropped the commit.